src/components/ItaliaTheme/Blocks/UOSearch/Sidebar.jsx

```diff
@@ -153,6 +158,21 @@ const Sidebar = (props) => {
               });
             }}
           />
+          <CheckboxWidget
+            id="always_show_image"
+            title={props.intl.formatMessage(messages.always_show_image)}
+            value={
+              props.data.always_show_image
+                ? props.data.always_show_image
+                : false
+            }
```

Comment:
Questo ternario e' un po' bruttino e secondo me non ti serve: https://github.com/plone/volto/blob/2d479891bd3cbafa95c3b3db0cf28102e171624e/packages/volto/src/components/manage/Widgets/CheckboxWidget.jsx#L27
Come vedi qui in volto hai gia' 
```
    <Checkbox
        name={`field-${id}`}
        checked={value || false}
```
Quindi anche un undefined/null viene gia' gestito, puoi passare quella prop liscia secondo me